internal/model/model.go

```diff
@@ -78,9 +99,11 @@ type Model struct {
 	detectedPackageName string
 	requestedFiles      map[string]bool
 	filteredLinesByFile map[string][]int
+    profilesLoaded      []*cover.Profile
```

Comment:
What do you think about calling this `coverProfiles` or `loadedProfiles` instead?